feature-libs/user/profile/components/update-email/update-email.module.ts

```diff
@@ -16,18 +16,45 @@ import {
   GlobalMessageService,
   I18nModule,
   provideDefaultConfig,
+  provideDefaultConfigFactory,
   RoutingService,
   UrlModule,
 } from '@spartacus/core';
 import {
   FormErrorsModule,
   SpinnerModule,
   PasswordVisibilityToggleModule,
+  MessageComponentModule,
 } from '@spartacus/storefront';
 import { UserEmailFacade } from '@spartacus/user/profile/root';
 import { UpdateEmailComponentService } from './update-email-component.service';
 import { UpdateEmailComponent } from './update-email.component';
 
+import { USE_MY_ACCOUNT_V2_EMAIL } from './use-my-account-v2-email.ts';
+import { MyAccountV2EmailComponent } from './my-account-v2-email.component';
+
+const myAccountV2EmailMapping: CmsConfig = {
+  cmsComponents: {
+    UpdateEmailComponent: {
+      component: MyAccountV2EmailComponent,
+      guards: [AuthGuard],
+      providers: [
+        {
+          provide: UpdateEmailComponentService,
+          useClass: UpdateEmailComponentService,
+          deps: [
+            UserEmailFacade,
+            RoutingService,
+            GlobalMessageService,
+            AuthService,
+            AuthRedirectService,
+          ],
+        },
+      ],
```

Comment:
[MINOR]
Do we really need to re-define `providers` in the cms mapping of v2? Since they are now identical to the `providers` in the cms mapping v1, I believe this can be dropped.

Note: analogical observation applies to also other NgModules in this PR (where parts of cms mappings identical in v1 and v2 don't need to be repeated in v2).
Note: similar observation applies also to property `guards` - it is also "inherited" from the cms mapping v1, and don't need to be re-declared in v2 if it's identical to v2

Reply:
Fixed